plugins/headless/hu.bme.mit.gamma.headless.server/src/hu/bme/mit/gamma/headless/server/util/IsDirectoryFilter.java

```diff
@@ -0,0 +1,17 @@
+package hu.bme.mit.gamma.headless.server.util;
+
+import java.io.File;
+import java.io.FileFilter;
+
+public class IsDirectoryFilter implements FileFilter {
+	
+	@Override
+	public boolean accept(File file) {
+		return file.isDirectory();
+	}
+	
+	public static IsDirectoryFilter Create() {
```

Comment:
Why is the first character of the method name capitalized?

Reply:
Also, couldn't this class be a singleton?

Reply:
fixed in 82e3b4f